services/artists-api/src/models/Artist/index.js

```diff
@@ -1,42 +1,35 @@
+// TODO: change social to an array
+// TODO add: approved: { type: Boolean, default: false, required: true },
+
 const mongoose = require('mongoose');
 const mongodbClient = require('../mongodbClient');
 
 let Artist = null;
 
 const schemaOptions = {
-  timestamps: true
+  timestamps: true,
 };
 const schema = new mongoose.Schema(
   {
+    approveStatus: {
+      type: String,
+      enum: ['approved', 'rejected', 'pending'],
+      default: 'pending',
+    },
+    reviewToken: { type: String, unique: true, required: true }, // TODO does it it unique option?
```

Comment:
i think it should be true because it reality we give it a value the moment we create an artist. If you think of a case we have to delete the value, let me know!